Emoji Art/EmojiArt.swift

```diff
@@ -30,6 +30,39 @@ struct EmojiArt {
         )
     }
     
+    mutating func removeEmoji(_ emoji: Emoji) {
+        if let index = emojis.firstIndex(where: { $0.id == emoji.id }) {
+            emojis.remove(at: index)
+        }
+    }
+    
+    subscript(_ emojiId: Emoji.ID) -> Emoji? {
+        if let index = index(of: emojiId) {
+            return emojis[index]
+        } else {
+            return nil
+        }
+    }
```

Comment:
A more swifty way to write this would be using the guard syntax:
```swift
    subscript(_ emojiId: Emoji.ID) -> Emoji? {
       guard let index = index(of: emojiId) else {
            return nil
       }
       return emojis[index]
    }